2020-05-08 11:43:44,507 p=72027 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-05-08 11:45:22,899 p=72027 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-05-08 11:45:22,925 p=72027 u=nisafajriyati |  TASK [Check if user specified in config is installed] **************************
2020-05-08 11:45:22,926 p=72027 u=nisafajriyati |  Friday 08 May 2020  11:45:22 +0800 (0:00:00.143)       0:00:00.143 ************ 
2020-05-08 11:45:25,677 p=72027 u=nisafajriyati |  fatal: [default -> localhost]: FAILED! => {"changed": false, "cmd": ["ssh", "-q", "-o", "BatchMode=yes", "-o", "ConnectTimeout=3", "default", "echo ok"], "delta": "0:00:02.273795", "end": "2020-05-08 11:45:25.642891", "msg": "non-zero return code", "rc": 255, "start": "2020-05-08 11:45:23.369096", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
2020-05-08 11:45:25,678 p=72027 u=nisafajriyati |  ...ignoring
2020-05-08 11:45:25,688 p=72027 u=nisafajriyati |  TASK [Install python2] *********************************************************
2020-05-08 11:45:25,688 p=72027 u=nisafajriyati |  Friday 08 May 2020  11:45:25 +0800 (0:00:02.762)       0:00:02.905 ************ 
2020-05-08 11:45:25,720 p=72027 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-05-08 11:45:30,904 p=72027 u=nisafajriyati |  fatal: [default]: FAILED! => {"changed": true, "msg": "non-zero return code", "rc": 100, "stderr": "Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.\r\nShared connection to 127.0.0.1 closed.\r\n", "stderr_lines": ["Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.", "Shared connection to 127.0.0.1 closed."], "stdout": "\rReading package lists... 0%\r\rReading package lists... 100%\r\rReading package lists... Done\r\r\n\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree       \r\r\n\rReading state information... 0%\r\rReading state information... 0%\r\rReading state information... Done\r\r\nThe following additional packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7\r\n  python2.7-minimal\r\nSuggested packages:\r\n  python-doc python-tk python2.7-doc binutils binfmt-support\r\nThe following NEW packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python\r\n  python-minimal python2.7 python2.7-minimal\r\n0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.\r\nNeed to get 3968 kB of archives.\r\nAfter this operation, 16.8 MB of additional disk space will be used.\r\n\u001b[33m\r0% [Working]\u001b[0m\u001b[33m\r0% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m\r0% [Connecting to archive.ubuntu.com (91.189.88.142)]\u001b[0m\r                                                     \rErr:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.152 80]\r\n\u001b[33m\r                                                     \r0% [Working]\u001b[0m\r            \rErr:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.152 80]\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rGet:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-minimal amd64 2.7.15~rc1-1 [28.1 kB]\r\n\u001b[33m\r0% [3 python-minimal 0 B/28.1 kB 0%]\u001b[0m\u001b[33m\r0% [3 python-minimal 12.9 kB/28.1 kB 46%]\u001b[0m\u001b[33m\r                                         \r3% [Working]\u001b[0m\r            \rErr:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.152 80]\r\n\u001b[33m\r3% [Working]\u001b[0m\r            \rErr:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.152 80]\r\n\u001b[33m\r3% [Working]\u001b[0m\r            \rGet:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-stdlib amd64 2.7.15~rc1-1 [7620 B]\r\n\u001b[33m\r3% [6 libpython-stdlib 0 B/7620 B 0%]\u001b[0m\u001b[33m\r                                     \r6% [Working]\u001b[0m\r            \rGet:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python amd64 2.7.15~rc1-1 [140 kB]\r\n\u001b[33m\r6% [7 python 0 B/140 kB 0%]\u001b[0m\u001b[33m\r7% [7 python 17.7 kB/140 kB 13%]\u001b[0m\u001b[33m\r                                \r9% [7 python 121 kB/140 kB 86%]\u001b[0m\u001b[33m\r                               \r12% [Working]\u001b[0m\r             \rFetched 176 kB in 4s (42.5 kB/s)\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]\r\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\r\n", "stdout_lines": ["", "Reading package lists... 0%", "", "Reading package lists... 100%", "", "Reading package lists... Done", "", "", "Building dependency tree... 0%", "", "Building dependency tree... 0%", "", "Building dependency tree... 50%", "", "Building dependency tree... 50%", "", "Building dependency tree       ", "", "", "Reading state information... 0%", "", "Reading state information... 0%", "", "Reading state information... Done", "", "The following additional packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7", "  python2.7-minimal", "Suggested packages:", "  python-doc python-tk python2.7-doc binutils binfmt-support", "The following NEW packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python", "  python-minimal python2.7 python2.7-minimal", "0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.", "Need to get 3968 kB of archives.", "After this operation, 16.8 MB of additional disk space will be used.", "\u001b[33m", "0% [Working]\u001b[0m\u001b[33m", "0% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m", "0% [Connecting to archive.ubuntu.com (91.189.88.142)]\u001b[0m", "                                                     ", "Err:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.152 80]", "\u001b[33m", "                                                     ", "0% [Working]\u001b[0m", "            ", "Err:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.152 80]", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Get:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-minimal amd64 2.7.15~rc1-1 [28.1 kB]", "\u001b[33m", "0% [3 python-minimal 0 B/28.1 kB 0%]\u001b[0m\u001b[33m", "0% [3 python-minimal 12.9 kB/28.1 kB 46%]\u001b[0m\u001b[33m", "                                         ", "3% [Working]\u001b[0m", "            ", "Err:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.152 80]", "\u001b[33m", "3% [Working]\u001b[0m", "            ", "Err:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.152 80]", "\u001b[33m", "3% [Working]\u001b[0m", "            ", "Get:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-stdlib amd64 2.7.15~rc1-1 [7620 B]", "\u001b[33m", "3% [6 libpython-stdlib 0 B/7620 B 0%]\u001b[0m\u001b[33m", "                                     ", "6% [Working]\u001b[0m", "            ", "Get:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python amd64 2.7.15~rc1-1 [140 kB]", "\u001b[33m", "6% [7 python 0 B/140 kB 0%]\u001b[0m\u001b[33m", "7% [7 python 17.7 kB/140 kB 13%]\u001b[0m\u001b[33m", "                                ", "9% [7 python 121 kB/140 kB 86%]\u001b[0m\u001b[33m", "                               ", "12% [Working]\u001b[0m", "             ", "Fetched 176 kB in 4s (42.5 kB/s)", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]", "E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?"]}
2020-05-08 11:45:30,905 p=72027 u=nisafajriyati |  PLAY RECAP *********************************************************************
2020-05-08 11:45:30,905 p=72027 u=nisafajriyati |  default                    : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=1   
2020-05-08 11:45:30,906 p=72027 u=nisafajriyati |  Friday 08 May 2020  11:45:30 +0800 (0:00:05.217)       0:00:08.123 ************ 
2020-05-08 11:45:30,906 p=72027 u=nisafajriyati |  =============================================================================== 
2020-05-08 11:45:30,906 p=72027 u=nisafajriyati |  Install python2 --------------------------------------------------------- 5.22s
2020-05-08 11:45:30,906 p=72027 u=nisafajriyati |  Check if user specified in config is installed -------------------------- 2.76s
2020-05-08 11:47:06,219 p=72528 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-05-08 11:47:13,703 p=72528 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-05-08 11:47:13,728 p=72528 u=nisafajriyati |  TASK [Check if user specified in config is installed] **************************
2020-05-08 11:47:13,728 p=72528 u=nisafajriyati |  Friday 08 May 2020  11:47:13 +0800 (0:00:00.136)       0:00:00.136 ************ 
2020-05-08 11:47:14,129 p=72528 u=nisafajriyati |  fatal: [default -> localhost]: FAILED! => {"changed": false, "cmd": ["ssh", "-q", "-o", "BatchMode=yes", "-o", "ConnectTimeout=3", "default", "echo ok"], "delta": "0:00:00.012125", "end": "2020-05-08 11:47:14.098708", "msg": "non-zero return code", "rc": 255, "start": "2020-05-08 11:47:14.086583", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
2020-05-08 11:47:14,129 p=72528 u=nisafajriyati |  ...ignoring
2020-05-08 11:47:14,137 p=72528 u=nisafajriyati |  TASK [Install python2] *********************************************************
2020-05-08 11:47:14,137 p=72528 u=nisafajriyati |  Friday 08 May 2020  11:47:14 +0800 (0:00:00.408)       0:00:00.545 ************ 
2020-05-08 11:47:14,159 p=72528 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-05-08 11:47:17,245 p=72528 u=nisafajriyati |  fatal: [default]: FAILED! => {"changed": true, "msg": "non-zero return code", "rc": 100, "stderr": "Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.\r\nShared connection to 127.0.0.1 closed.\r\n", "stderr_lines": ["Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.", "Shared connection to 127.0.0.1 closed."], "stdout": "\rReading package lists... 0%\r\rReading package lists... 100%\r\rReading package lists... Done\r\r\n\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree       \r\r\n\rReading state information... 0%\r\rReading state information... 0%\r\rReading state information... Done\r\r\nThe following additional packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7\r\n  python2.7-minimal\r\nSuggested packages:\r\n  python-doc python-tk python2.7-doc binutils binfmt-support\r\nThe following NEW packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python\r\n  python-minimal python2.7 python2.7-minimal\r\n0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.\r\nNeed to get 3792 kB/3968 kB of archives.\r\nAfter this operation, 16.8 MB of additional disk space will be used.\r\n\u001b[33m\r12% [Working]\u001b[0m\u001b[33m\r12% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m\r                                      \r12% [Waiting for headers]\u001b[0m\r                         \rErr:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.142 80]\r\n\u001b[33m\r                         \r12% [Working]\u001b[0m\u001b[33m\r12% [Waiting for headers]\u001b[0m\r                         \rErr:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.142 80]\r\n\u001b[33m\r12% [Waiting for headers]\u001b[0m\r                         \rErr:3 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.142 80]\r\n\u001b[33m\r12% [Waiting for headers]\u001b[0m\r                         \rErr:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.142 80]\r\n\u001b[33m\r                         \r12% [Working]\u001b[0m\r             \rE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.142 80]\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.142 80]\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.142 80]\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.142 80]\r\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\r\n", "stdout_lines": ["", "Reading package lists... 0%", "", "Reading package lists... 100%", "", "Reading package lists... Done", "", "", "Building dependency tree... 0%", "", "Building dependency tree... 0%", "", "Building dependency tree... 50%", "", "Building dependency tree... 50%", "", "Building dependency tree       ", "", "", "Reading state information... 0%", "", "Reading state information... 0%", "", "Reading state information... Done", "", "The following additional packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7", "  python2.7-minimal", "Suggested packages:", "  python-doc python-tk python2.7-doc binutils binfmt-support", "The following NEW packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python", "  python-minimal python2.7 python2.7-minimal", "0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.", "Need to get 3792 kB/3968 kB of archives.", "After this operation, 16.8 MB of additional disk space will be used.", "\u001b[33m", "12% [Working]\u001b[0m\u001b[33m", "12% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m", "                                      ", "12% [Waiting for headers]\u001b[0m", "                         ", "Err:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.142 80]", "\u001b[33m", "                         ", "12% [Working]\u001b[0m\u001b[33m", "12% [Waiting for headers]\u001b[0m", "                         ", "Err:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.142 80]", "\u001b[33m", "12% [Waiting for headers]\u001b[0m", "                         ", "Err:3 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.142 80]", "\u001b[33m", "12% [Waiting for headers]\u001b[0m", "                         ", "Err:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.142 80]", "\u001b[33m", "                         ", "12% [Working]\u001b[0m", "             ", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.142 80]", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.142 80]", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.142 80]", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.142 80]", "E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?"]}
2020-05-08 11:47:17,247 p=72528 u=nisafajriyati |  PLAY RECAP *********************************************************************
2020-05-08 11:47:17,247 p=72528 u=nisafajriyati |  default                    : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=1   
2020-05-08 11:47:17,247 p=72528 u=nisafajriyati |  Friday 08 May 2020  11:47:17 +0800 (0:00:03.109)       0:00:03.655 ************ 
2020-05-08 11:47:17,247 p=72528 u=nisafajriyati |  =============================================================================== 
2020-05-08 11:47:17,247 p=72528 u=nisafajriyati |  Install python2 --------------------------------------------------------- 3.11s
2020-05-08 11:47:17,247 p=72528 u=nisafajriyati |  Check if user specified in config is installed -------------------------- 0.41s
2020-05-08 11:49:58,836 p=72933 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-05-08 11:50:05,829 p=72933 u=nisafajriyati |  ERROR! the role 'kamaln7.swapfile' was not found in /Users/nisafajriyati/Documents/College/TA/vm-myits/setup-TA/roles:/Users/nisafajriyati/.ansible/roles:/etc/ansible/roles:/Users/nisafajriyati/Documents/College/TA/vm-myits/setup-TA

The error appears to be in '/Users/nisafajriyati/Documents/College/TA/vm-myits/setup-TA/setup.yml': line 22, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


    - name: kamaln7.swapfile
      ^ here

2020-05-08 11:50:30,264 p=73037 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-05-08 11:50:43,265 p=73037 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-05-08 11:50:43,284 p=73037 u=nisafajriyati |  TASK [Gathering Facts] *********************************************************
2020-05-08 11:50:43,284 p=73037 u=nisafajriyati |  Friday 08 May 2020  11:50:43 +0800 (0:00:00.109)       0:00:00.109 ************ 
2020-05-08 11:50:45,084 p=73037 u=nisafajriyati |  ok: [default]
2020-05-08 11:50:45,117 p=73037 u=nisafajriyati |  TASK [Get current build number] ************************************************
2020-05-08 11:50:45,117 p=73037 u=nisafajriyati |  Friday 08 May 2020  11:50:45 +0800 (0:00:01.832)       0:00:01.942 ************ 
2020-05-08 11:50:45,542 p=73037 u=nisafajriyati |  changed: [default -> localhost]
2020-05-08 11:50:45,563 p=73037 u=nisafajriyati |  TASK [Register release variables] **********************************************
2020-05-08 11:50:45,563 p=73037 u=nisafajriyati |  Friday 08 May 2020  11:50:45 +0800 (0:00:00.446)       0:00:02.388 ************ 
2020-05-08 11:50:45,599 p=73037 u=nisafajriyati |  ok: [default]
2020-05-08 11:50:45,621 p=73037 u=nisafajriyati |  TASK [Get all releases dir] ****************************************************
2020-05-08 11:50:45,621 p=73037 u=nisafajriyati |  Friday 08 May 2020  11:50:45 +0800 (0:00:00.057)       0:00:02.446 ************ 
2020-05-08 11:50:46,022 p=73037 u=nisafajriyati |  ok: [default]
2020-05-08 11:50:46,036 p=73037 u=nisafajriyati |  TASK [Register latest release] *************************************************
2020-05-08 11:50:46,037 p=73037 u=nisafajriyati |  Friday 08 May 2020  11:50:46 +0800 (0:00:00.415)       0:00:02.861 ************ 
2020-05-08 11:50:46,061 p=73037 u=nisafajriyati |  skipping: [default]
2020-05-08 11:50:46,080 p=73037 u=nisafajriyati |  TASK [Create new release directory] ********************************************
2020-05-08 11:50:46,081 p=73037 u=nisafajriyati |  Friday 08 May 2020  11:50:46 +0800 (0:00:00.044)       0:00:02.905 ************ 
2020-05-08 11:50:46,510 p=73037 u=nisafajriyati |  changed: [default]
2020-05-08 11:50:46,530 p=73037 u=nisafajriyati |  TASK [Clone repo] **************************************************************
2020-05-08 11:50:46,530 p=73037 u=nisafajriyati |  Friday 08 May 2020  11:50:46 +0800 (0:00:00.449)       0:00:03.355 ************ 
2020-05-08 12:41:24,841 p=73037 u=nisafajriyati |   [ERROR]: User interrupted execution

2020-05-08 12:50:02,431 p=80624 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-05-08 12:50:09,848 p=80624 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-05-08 12:50:09,871 p=80624 u=nisafajriyati |  TASK [Check if user specified in config is installed] **************************
2020-05-08 12:50:09,871 p=80624 u=nisafajriyati |  Friday 08 May 2020  12:50:09 +0800 (0:00:00.138)       0:00:00.138 ************ 
2020-05-08 12:50:10,498 p=80624 u=nisafajriyati |  fatal: [repo -> localhost]: FAILED! => {"changed": false, "cmd": ["ssh", "-q", "-o", "BatchMode=yes", "-o", "ConnectTimeout=3", "repo", "echo ok"], "delta": "0:00:00.215287", "end": "2020-05-08 12:50:10.460889", "msg": "non-zero return code", "rc": 255, "start": "2020-05-08 12:50:10.245602", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
2020-05-08 12:50:10,498 p=80624 u=nisafajriyati |  ...ignoring
2020-05-08 12:50:10,510 p=80624 u=nisafajriyati |  TASK [Install python2] *********************************************************
2020-05-08 12:50:10,510 p=80624 u=nisafajriyati |  Friday 08 May 2020  12:50:10 +0800 (0:00:00.638)       0:00:00.777 ************ 
2020-05-08 12:50:10,537 p=80624 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-05-08 12:50:19,069 p=80624 u=nisafajriyati |  fatal: [repo]: FAILED! => {"changed": true, "msg": "non-zero return code", "rc": 100, "stderr": "Warning: Permanently added '[127.0.0.1]:2200' (ECDSA) to the list of known hosts.\r\nShared connection to 127.0.0.1 closed.\r\n", "stderr_lines": ["Warning: Permanently added '[127.0.0.1]:2200' (ECDSA) to the list of known hosts.", "Shared connection to 127.0.0.1 closed."], "stdout": "\rReading package lists... 0%\r\rReading package lists... 100%\r\rReading package lists... Done\r\r\n\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree       \r\r\n\rReading state information... 0%\r\rReading state information... 0%\r\rReading state information... Done\r\r\nThe following additional packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7\r\n  python2.7-minimal\r\nSuggested packages:\r\n  python-doc python-tk python2.7-doc binutils binfmt-support\r\nThe following NEW packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python\r\n  python-minimal python2.7 python2.7-minimal\r\n0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.\r\nNeed to get 3968 kB of archives.\r\nAfter this operation, 16.8 MB of additional disk space will be used.\r\n\u001b[33m\r0% [Working]\u001b[0m\u001b[33m\r0% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m\r0% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m\r0% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m\r0% [Connecting to archive.ubuntu.com (91.189.88.142)]\u001b[0m\u001b[33m\r0% [Connecting to archive.ubuntu.com (2001:67c:1562::15)]\u001b[0m\r                                                         \rErr:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.152 80]\r\n\u001b[33m\r                                                         \r0% [Waiting for headers]\u001b[0m\r                        \rErr:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.152 80]\r\n\u001b[33m\r0% [Waiting for headers]\u001b[0m\r                        \rGet:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-minimal amd64 2.7.15~rc1-1 [28.1 kB]\r\n\u001b[33m\r0% [3 python-minimal 326 B/28.1 kB 1%]\u001b[0m\u001b[33m\r0% [3 python-minimal 13.9 kB/28.1 kB 50%]\u001b[0m\u001b[33m\r                                         \r3% [Working]\u001b[0m\r            \rErr:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.152 80]\r\n\u001b[33m\r3% [Working]\u001b[0m\r            \rErr:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04\r\n  404  Not Found [IP: 91.189.88.152 80]\r\n\u001b[33m\r3% [Working]\u001b[0m\r            \rGet:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-stdlib amd64 2.7.15~rc1-1 [7620 B]\r\n\u001b[33m\r3% [6 libpython-stdlib 0 B/7620 B 0%]\u001b[0m\u001b[33m\r                                     \r6% [Working]\u001b[0m\r            \rGet:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python amd64 2.7.15~rc1-1 [140 kB]\r\n\u001b[33m\r6% [7 python 0 B/140 kB 0%]\u001b[0m\u001b[33m\r7% [7 python 11.4 kB/140 kB 8%]\u001b[0m\u001b[33m\r8% [7 python 80.6 kB/140 kB 58%]\u001b[0m\u001b[33m\r8% [7 python 80.6 kB/140 kB 58%]                             19.3 kB/s 3min 19s\u001b[0m\u001b[33m\r8% [7 python 80.6 kB/140 kB 58%]                             19.3 kB/s 3min 19s\u001b[0m\u001b[33m\r8% [7 python 80.6 kB/140 kB 58%]                             19.3 kB/s 3min 19s\u001b[0m\u001b[33m\r12% [Working]                                                19.3 kB/s 3min 16s\u001b[0m\r                                                                               \rFetched 176 kB in 7s (24.2 kB/s)\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]\r\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\r\n", "stdout_lines": ["", "Reading package lists... 0%", "", "Reading package lists... 100%", "", "Reading package lists... Done", "", "", "Building dependency tree... 0%", "", "Building dependency tree... 0%", "", "Building dependency tree... 50%", "", "Building dependency tree... 50%", "", "Building dependency tree       ", "", "", "Reading state information... 0%", "", "Reading state information... 0%", "", "Reading state information... Done", "", "The following additional packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7", "  python2.7-minimal", "Suggested packages:", "  python-doc python-tk python2.7-doc binutils binfmt-support", "The following NEW packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python", "  python-minimal python2.7 python2.7-minimal", "0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.", "Need to get 3968 kB of archives.", "After this operation, 16.8 MB of additional disk space will be used.", "\u001b[33m", "0% [Working]\u001b[0m\u001b[33m", "0% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m", "0% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m", "0% [Connecting to archive.ubuntu.com]\u001b[0m\u001b[33m", "0% [Connecting to archive.ubuntu.com (91.189.88.142)]\u001b[0m\u001b[33m", "0% [Connecting to archive.ubuntu.com (2001:67c:1562::15)]\u001b[0m", "                                                         ", "Err:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.152 80]", "\u001b[33m", "                                                         ", "0% [Waiting for headers]\u001b[0m", "                        ", "Err:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.152 80]", "\u001b[33m", "0% [Waiting for headers]\u001b[0m", "                        ", "Get:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-minimal amd64 2.7.15~rc1-1 [28.1 kB]", "\u001b[33m", "0% [3 python-minimal 326 B/28.1 kB 1%]\u001b[0m\u001b[33m", "0% [3 python-minimal 13.9 kB/28.1 kB 50%]\u001b[0m\u001b[33m", "                                         ", "3% [Working]\u001b[0m", "            ", "Err:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.152 80]", "\u001b[33m", "3% [Working]\u001b[0m", "            ", "Err:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04", "  404  Not Found [IP: 91.189.88.152 80]", "\u001b[33m", "3% [Working]\u001b[0m", "            ", "Get:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-stdlib amd64 2.7.15~rc1-1 [7620 B]", "\u001b[33m", "3% [6 libpython-stdlib 0 B/7620 B 0%]\u001b[0m\u001b[33m", "                                     ", "6% [Working]\u001b[0m", "            ", "Get:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python amd64 2.7.15~rc1-1 [140 kB]", "\u001b[33m", "6% [7 python 0 B/140 kB 0%]\u001b[0m\u001b[33m", "7% [7 python 11.4 kB/140 kB 8%]\u001b[0m\u001b[33m", "8% [7 python 80.6 kB/140 kB 58%]\u001b[0m\u001b[33m", "8% [7 python 80.6 kB/140 kB 58%]                             19.3 kB/s 3min 19s\u001b[0m\u001b[33m", "8% [7 python 80.6 kB/140 kB 58%]                             19.3 kB/s 3min 19s\u001b[0m\u001b[33m", "8% [7 python 80.6 kB/140 kB 58%]                             19.3 kB/s 3min 19s\u001b[0m\u001b[33m", "12% [Working]                                                19.3 kB/s 3min 16s\u001b[0m", "                                                                               ", "Fetched 176 kB in 7s (24.2 kB/s)", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  404  Not Found [IP: 91.189.88.152 80]", "E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?"]}
2020-05-08 12:50:19,071 p=80624 u=nisafajriyati |  PLAY RECAP *********************************************************************
2020-05-08 12:50:19,072 p=80624 u=nisafajriyati |  repo                       : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=1   
2020-05-08 12:50:19,072 p=80624 u=nisafajriyati |  Friday 08 May 2020  12:50:19 +0800 (0:00:08.561)       0:00:09.338 ************ 
2020-05-08 12:50:19,072 p=80624 u=nisafajriyati |  =============================================================================== 
2020-05-08 12:50:19,072 p=80624 u=nisafajriyati |  Install python2 --------------------------------------------------------- 8.56s
2020-05-08 12:50:19,072 p=80624 u=nisafajriyati |  Check if user specified in config is installed -------------------------- 0.64s
2020-05-08 12:52:00,798 p=80937 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-05-08 12:52:07,624 p=80937 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-05-08 12:52:07,645 p=80937 u=nisafajriyati |  TASK [Gathering Facts] *********************************************************
2020-05-08 12:52:07,645 p=80937 u=nisafajriyati |  Friday 08 May 2020  12:52:07 +0800 (0:00:00.112)       0:00:00.112 ************ 
2020-05-08 12:52:09,361 p=80937 u=nisafajriyati |  ok: [repo]
2020-05-08 12:52:09,389 p=80937 u=nisafajriyati |  TASK [Get current build number] ************************************************
2020-05-08 12:52:09,390 p=80937 u=nisafajriyati |  Friday 08 May 2020  12:52:09 +0800 (0:00:01.744)       0:00:01.857 ************ 
2020-05-08 12:52:09,824 p=80937 u=nisafajriyati |  changed: [repo -> localhost]
2020-05-08 12:52:09,840 p=80937 u=nisafajriyati |  TASK [Register release variables] **********************************************
2020-05-08 12:52:09,840 p=80937 u=nisafajriyati |  Friday 08 May 2020  12:52:09 +0800 (0:00:00.450)       0:00:02.307 ************ 
2020-05-08 12:52:09,871 p=80937 u=nisafajriyati |  ok: [repo]
2020-05-08 12:52:09,889 p=80937 u=nisafajriyati |  TASK [Get all releases dir] ****************************************************
2020-05-08 12:52:09,889 p=80937 u=nisafajriyati |  Friday 08 May 2020  12:52:09 +0800 (0:00:00.049)       0:00:02.356 ************ 
2020-05-08 12:52:10,278 p=80937 u=nisafajriyati |  ok: [repo]
2020-05-08 12:52:10,294 p=80937 u=nisafajriyati |  TASK [Register latest release] *************************************************
2020-05-08 12:52:10,295 p=80937 u=nisafajriyati |  Friday 08 May 2020  12:52:10 +0800 (0:00:00.405)       0:00:02.762 ************ 
2020-05-08 12:52:10,319 p=80937 u=nisafajriyati |  skipping: [repo]
2020-05-08 12:52:10,337 p=80937 u=nisafajriyati |  TASK [Create new release directory] ********************************************
2020-05-08 12:52:10,338 p=80937 u=nisafajriyati |  Friday 08 May 2020  12:52:10 +0800 (0:00:00.042)       0:00:02.805 ************ 
2020-05-08 12:52:10,774 p=80937 u=nisafajriyati |  changed: [repo]
2020-05-08 12:52:10,789 p=80937 u=nisafajriyati |  TASK [Clone repo] **************************************************************
2020-05-08 12:52:10,789 p=80937 u=nisafajriyati |  Friday 08 May 2020  12:52:10 +0800 (0:00:00.451)       0:00:03.256 ************ 
2020-05-08 17:08:10,224 p=80937 u=nisafajriyati |   [ERROR]: User interrupted execution

2020-06-07 11:42:31,572 p=2087 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-06-07 11:42:42,012 p=2087 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-06-07 11:42:42,033 p=2087 u=nisafajriyati |  TASK [Check if user specified in config is installed] **************************
2020-06-07 11:42:42,033 p=2087 u=nisafajriyati |  Sunday 07 June 2020  11:42:42 +0800 (0:00:00.137)       0:00:00.137 *********** 
2020-06-07 11:42:42,451 p=2087 u=nisafajriyati |  fatal: [repo -> localhost]: FAILED! => {"changed": false, "cmd": ["ssh", "-q", "-o", "BatchMode=yes", "-o", "ConnectTimeout=3", "repo", "echo ok"], "delta": "0:00:00.015089", "end": "2020-06-07 11:42:42.420844", "msg": "non-zero return code", "rc": 255, "start": "2020-06-07 11:42:42.405755", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
2020-06-07 11:42:42,451 p=2087 u=nisafajriyati |  ...ignoring
2020-06-07 11:42:42,458 p=2087 u=nisafajriyati |  TASK [Install python2] *********************************************************
2020-06-07 11:42:42,458 p=2087 u=nisafajriyati |  Sunday 07 June 2020  11:42:42 +0800 (0:00:00.424)       0:00:00.562 *********** 
2020-06-07 11:42:42,469 p=2087 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-07 11:42:43,464 p=2087 u=nisafajriyati |  fatal: [repo]: FAILED! => {"changed": true, "msg": "non-zero return code", "rc": 100, "stderr": "Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.\r\nShared connection to 127.0.0.1 closed.\r\n", "stderr_lines": ["Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.", "Shared connection to 127.0.0.1 closed."], "stdout": "\rReading package lists... 0%\r\rReading package lists... 100%\r\rReading package lists... Done\r\r\n\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree       \r\r\n\rReading state information... 0%\r\rReading state information... 0%\r\rReading state information... Done\r\r\nThe following additional packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7\r\n  python2.7-minimal\r\nSuggested packages:\r\n  python-doc python-tk python2.7-doc binutils binfmt-support\r\nThe following NEW packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python\r\n  python-minimal python2.7 python2.7-minimal\r\n0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.\r\nNeed to get 3968 kB of archives.\r\nAfter this operation, 16.8 MB of additional disk space will be used.\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-minimal amd64 2.7.15~rc1-1\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-stdlib amd64 2.7.15~rc1-1\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python amd64 2.7.15~rc1-1\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/python-minimal_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/libpython-stdlib_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/python_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\r\n", "stdout_lines": ["", "Reading package lists... 0%", "", "Reading package lists... 100%", "", "Reading package lists... Done", "", "", "Building dependency tree... 0%", "", "Building dependency tree... 0%", "", "Building dependency tree... 50%", "", "Building dependency tree... 50%", "", "Building dependency tree       ", "", "", "Reading state information... 0%", "", "Reading state information... 0%", "", "Reading state information... Done", "", "The following additional packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7", "  python2.7-minimal", "Suggested packages:", "  python-doc python-tk python2.7-doc binutils binfmt-support", "The following NEW packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python", "  python-minimal python2.7 python2.7-minimal", "0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.", "Need to get 3968 kB of archives.", "After this operation, 16.8 MB of additional disk space will be used.", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-minimal amd64 2.7.15~rc1-1", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-stdlib amd64 2.7.15~rc1-1", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python amd64 2.7.15~rc1-1", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/python-minimal_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/libpython-stdlib_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/python_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?"]}
2020-06-07 11:42:43,466 p=2087 u=nisafajriyati |  PLAY RECAP *********************************************************************
2020-06-07 11:42:43,466 p=2087 u=nisafajriyati |  repo                       : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=1   
2020-06-07 11:42:43,466 p=2087 u=nisafajriyati |  Sunday 07 June 2020  11:42:43 +0800 (0:00:01.008)       0:00:01.571 *********** 
2020-06-07 11:42:43,466 p=2087 u=nisafajriyati |  =============================================================================== 
2020-06-07 11:42:43,466 p=2087 u=nisafajriyati |  Install python2 --------------------------------------------------------- 1.01s
2020-06-07 11:42:43,466 p=2087 u=nisafajriyati |  Check if user specified in config is installed -------------------------- 0.42s
2020-06-07 11:45:46,257 p=2643 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-06-07 11:46:08,291 p=2643 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-06-07 11:46:08,318 p=2643 u=nisafajriyati |  TASK [Check if user specified in config is installed] **************************
2020-06-07 11:46:08,318 p=2643 u=nisafajriyati |  Sunday 07 June 2020  11:46:08 +0800 (0:00:00.137)       0:00:00.137 *********** 
2020-06-07 11:46:08,724 p=2643 u=nisafajriyati |  fatal: [repo -> localhost]: FAILED! => {"changed": false, "cmd": ["ssh", "-q", "-o", "BatchMode=yes", "-o", "ConnectTimeout=3", "repo", "echo ok"], "delta": "0:00:00.012612", "end": "2020-06-07 11:46:08.696533", "msg": "non-zero return code", "rc": 255, "start": "2020-06-07 11:46:08.683921", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
2020-06-07 11:46:08,724 p=2643 u=nisafajriyati |  ...ignoring
2020-06-07 11:46:08,732 p=2643 u=nisafajriyati |  TASK [Install python2] *********************************************************
2020-06-07 11:46:08,733 p=2643 u=nisafajriyati |  Sunday 07 June 2020  11:46:08 +0800 (0:00:00.414)       0:00:00.552 *********** 
2020-06-07 11:46:08,756 p=2643 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-07 11:46:09,712 p=2643 u=nisafajriyati |  fatal: [repo]: FAILED! => {"changed": true, "msg": "non-zero return code", "rc": 100, "stderr": "Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.\r\nShared connection to 127.0.0.1 closed.\r\n", "stderr_lines": ["Warning: Permanently added '[127.0.0.1]:2222' (ECDSA) to the list of known hosts.", "Shared connection to 127.0.0.1 closed."], "stdout": "\rReading package lists... 0%\r\rReading package lists... 100%\r\rReading package lists... Done\r\r\n\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 0%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree... 50%\r\rBuilding dependency tree       \r\r\n\rReading state information... 0%\r\rReading state information... 0%\r\rReading state information... Done\r\r\nThe following additional packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7\r\n  python2.7-minimal\r\nSuggested packages:\r\n  python-doc python-tk python2.7-doc binutils binfmt-support\r\nThe following NEW packages will be installed:\r\n  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python\r\n  python-minimal python2.7 python2.7-minimal\r\n0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.\r\nNeed to get 3968 kB of archives.\r\nAfter this operation, 16.8 MB of additional disk space will be used.\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-minimal amd64 2.7.15~rc1-1\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-stdlib amd64 2.7.15~rc1-1\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rErr:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python amd64 2.7.15~rc1-1\r\n  Could not resolve 'archive.ubuntu.com'\r\n\u001b[33m\r0% [Working]\u001b[0m\r            \rE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/python-minimal_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/libpython-stdlib_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/python_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'\r\nE: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?\r\n", "stdout_lines": ["", "Reading package lists... 0%", "", "Reading package lists... 100%", "", "Reading package lists... Done", "", "", "Building dependency tree... 0%", "", "Building dependency tree... 0%", "", "Building dependency tree... 50%", "", "Building dependency tree... 50%", "", "Building dependency tree       ", "", "", "Reading state information... 0%", "", "Reading state information... 0%", "", "Reading state information... Done", "", "The following additional packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python python2.7", "  python2.7-minimal", "Suggested packages:", "  python-doc python-tk python2.7-doc binutils binfmt-support", "The following NEW packages will be installed:", "  libpython-stdlib libpython2.7-minimal libpython2.7-stdlib python", "  python-minimal python2.7 python2.7-minimal", "0 upgraded, 7 newly installed, 0 to remove and 0 not upgraded.", "Need to get 3968 kB of archives.", "After this operation, 16.8 MB of additional disk space will be used.", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-minimal amd64 2.7.17-1~18.04", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7-minimal amd64 2.7.17-1~18.04", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-minimal amd64 2.7.15~rc1-1", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython2.7-stdlib amd64 2.7.17-1~18.04", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:5 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python2.7 amd64 2.7.17-1~18.04", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 libpython-stdlib amd64 2.7.15~rc1-1", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "Err:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python amd64 2.7.15~rc1-1", "  Could not resolve 'archive.ubuntu.com'", "\u001b[33m", "0% [Working]\u001b[0m", "            ", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-minimal_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7-minimal_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/python-minimal_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/libpython2.7-stdlib_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python2.7/python2.7_2.7.17-1~18.04_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/libpython-stdlib_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/p/python-defaults/python_2.7.15~rc1-1_amd64.deb  Could not resolve 'archive.ubuntu.com'", "E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?"]}
2020-06-07 11:46:09,713 p=2643 u=nisafajriyati |  PLAY RECAP *********************************************************************
2020-06-07 11:46:09,714 p=2643 u=nisafajriyati |  repo                       : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=1   
2020-06-07 11:46:09,714 p=2643 u=nisafajriyati |  Sunday 07 June 2020  11:46:09 +0800 (0:00:00.981)       0:00:01.533 *********** 
2020-06-07 11:46:09,714 p=2643 u=nisafajriyati |  =============================================================================== 
2020-06-07 11:46:09,714 p=2643 u=nisafajriyati |  Install python2 --------------------------------------------------------- 0.98s
2020-06-07 11:46:09,714 p=2643 u=nisafajriyati |  Check if user specified in config is installed -------------------------- 0.41s
2020-06-07 12:04:12,114 p=5040 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-06-07 12:04:16,297 p=5040 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-06-07 12:04:16,325 p=5040 u=nisafajriyati |  TASK [Check if user specified in config is installed] **************************
2020-06-07 12:04:16,325 p=5040 u=nisafajriyati |  Sunday 07 June 2020  12:04:16 +0800 (0:00:00.137)       0:00:00.137 *********** 
2020-06-07 12:04:17,172 p=5040 u=nisafajriyati |  fatal: [repo -> localhost]: FAILED! => {"changed": false, "cmd": ["ssh", "-q", "-o", "BatchMode=yes", "-o", "ConnectTimeout=3", "repo", "echo ok"], "delta": "0:00:00.395630", "end": "2020-06-07 12:04:17.138809", "msg": "non-zero return code", "rc": 255, "start": "2020-06-07 12:04:16.743179", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
2020-06-07 12:04:17,172 p=5040 u=nisafajriyati |  ...ignoring
2020-06-07 12:04:17,181 p=5040 u=nisafajriyati |  TASK [Install python2] *********************************************************
2020-06-07 12:04:17,181 p=5040 u=nisafajriyati |  Sunday 07 June 2020  12:04:17 +0800 (0:00:00.855)       0:00:00.993 *********** 
2020-06-07 12:04:17,198 p=5040 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-07 12:04:18,620 p=5040 u=nisafajriyati |  changed: [repo]
2020-06-07 12:04:18,630 p=5040 u=nisafajriyati |  TASK [Install aptitude] ********************************************************
2020-06-07 12:04:18,630 p=5040 u=nisafajriyati |  Sunday 07 June 2020  12:04:18 +0800 (0:00:01.448)       0:00:02.442 *********** 
2020-06-07 12:04:18,650 p=5040 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-07 12:05:01,348 p=5040 u=nisafajriyati |   [WARNING]: Updating cache and auto-installing missing dependency: python-apt

2020-06-07 12:05:01,349 p=5040 u=nisafajriyati |   [WARNING]: Could not find aptitude. Using apt-get instead

2020-06-07 12:05:01,349 p=5040 u=nisafajriyati |  [DEPRECATION WARNING]: Distribution Ubuntu 18.04 on host repo should use 
/usr/bin/python3, but is using /usr/bin/python for backward compatibility with 
prior Ansible releases. A future Ansible release will default to using the 
discovered platform python for this host. See https://docs.ansible.com/ansible/
2.8/reference_appendices/interpreter_discovery.html for more information. This 
feature will be removed in version 2.12. Deprecation warnings can be disabled 
by setting deprecation_warnings=False in ansible.cfg.
2020-06-07 12:05:01,350 p=5040 u=nisafajriyati |  changed: [repo]
2020-06-07 12:05:01,360 p=5040 u=nisafajriyati |  TASK [Update all packages] *****************************************************
2020-06-07 12:05:01,361 p=5040 u=nisafajriyati |  Sunday 07 June 2020  12:05:01 +0800 (0:00:42.730)       0:00:45.173 *********** 
2020-06-07 12:05:01,375 p=5040 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-07 12:10:08,415 p=5040 u=nisafajriyati |  changed: [repo]
2020-06-07 12:10:08,430 p=5040 u=nisafajriyati |  TASK [user : Ensure sudo group is present] *************************************
2020-06-07 12:10:08,431 p=5040 u=nisafajriyati |  Sunday 07 June 2020  12:10:08 +0800 (0:05:07.069)       0:05:52.243 *********** 
2020-06-07 12:10:08,452 p=5040 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-07 12:10:08,891 p=5040 u=nisafajriyati |  ok: [repo]
2020-06-07 12:10:08,899 p=5040 u=nisafajriyati |  TASK [user : Ensure sudo group has sudo privileges] ****************************
2020-06-07 12:10:08,899 p=5040 u=nisafajriyati |  Sunday 07 June 2020  12:10:08 +0800 (0:00:00.468)       0:05:52.711 *********** 
2020-06-07 12:10:08,917 p=5040 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-07 12:10:09,255 p=5040 u=nisafajriyati |  changed: [repo]
2020-06-07 12:10:09,263 p=5040 u=nisafajriyati |  TASK [user : Add new super user] ***********************************************
2020-06-07 12:10:09,263 p=5040 u=nisafajriyati |  Sunday 07 June 2020  12:10:09 +0800 (0:00:00.364)       0:05:53.075 *********** 
2020-06-07 12:10:09,280 p=5040 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-07 12:10:09,797 p=5040 u=nisafajriyati |  changed: [repo]
2020-06-07 12:10:09,805 p=5040 u=nisafajriyati |  TASK [user : Add ssh key for new super user] ***********************************
2020-06-07 12:10:09,805 p=5040 u=nisafajriyati |  Sunday 07 June 2020  12:10:09 +0800 (0:00:00.542)       0:05:53.617 *********** 
2020-06-07 12:10:09,843 p=5040 u=nisafajriyati |   [WARNING]: Unable to find '~/.ssh/authorized_keys' in expected paths (use
-vvvvv to see paths)

2020-06-07 12:10:09,848 p=5040 u=nisafajriyati |  fatal: [repo]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'file'. Error was a <class 'ansible.errors.AnsibleError'>, original message: could not locate file in lookup: ~/.ssh/authorized_keys"}
2020-06-07 12:10:09,849 p=5040 u=nisafajriyati |  PLAY RECAP *********************************************************************
2020-06-07 12:10:09,850 p=5040 u=nisafajriyati |  repo                       : ok=7    changed=5    unreachable=0    failed=1    skipped=0    rescued=0    ignored=1   
2020-06-07 12:10:09,850 p=5040 u=nisafajriyati |  Sunday 07 June 2020  12:10:09 +0800 (0:00:00.044)       0:05:53.662 *********** 
2020-06-07 12:10:09,850 p=5040 u=nisafajriyati |  =============================================================================== 
2020-06-07 12:10:09,850 p=5040 u=nisafajriyati |  Update all packages --------------------------------------------------- 307.07s
2020-06-07 12:10:09,850 p=5040 u=nisafajriyati |  Install aptitude ------------------------------------------------------- 42.73s
2020-06-07 12:10:09,850 p=5040 u=nisafajriyati |  Install python2 --------------------------------------------------------- 1.45s
2020-06-07 12:10:09,851 p=5040 u=nisafajriyati |  Check if user specified in config is installed -------------------------- 0.86s
2020-06-07 12:10:09,851 p=5040 u=nisafajriyati |  user : Add new super user ----------------------------------------------- 0.54s
2020-06-07 12:10:09,851 p=5040 u=nisafajriyati |  user : Ensure sudo group is present ------------------------------------- 0.47s
2020-06-07 12:10:09,851 p=5040 u=nisafajriyati |  user : Ensure sudo group has sudo privileges ---------------------------- 0.36s
2020-06-07 12:10:09,851 p=5040 u=nisafajriyati |  user : Add ssh key for new super user ----------------------------------- 0.04s
2020-06-08 11:15:12,745 p=24602 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-06-08 11:21:19,364 p=24602 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-06-08 11:21:19,397 p=24602 u=nisafajriyati |  TASK [Check if user specified in config is installed] **************************
2020-06-08 11:21:19,397 p=24602 u=nisafajriyati |  Monday 08 June 2020  11:21:19 +0800 (0:00:00.143)       0:00:00.143 *********** 
2020-06-08 11:21:20,196 p=24602 u=nisafajriyati |  fatal: [default -> localhost]: FAILED! => {"changed": false, "cmd": ["ssh", "-q", "-o", "BatchMode=yes", "-o", "ConnectTimeout=3", "default", "echo ok"], "delta": "0:00:00.317567", "end": "2020-06-08 11:21:20.164711", "msg": "non-zero return code", "rc": 255, "start": "2020-06-08 11:21:19.847144", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
2020-06-08 11:21:20,197 p=24602 u=nisafajriyati |  ...ignoring
2020-06-08 11:21:20,204 p=24602 u=nisafajriyati |  TASK [Install python2] *********************************************************
2020-06-08 11:21:20,204 p=24602 u=nisafajriyati |  Monday 08 June 2020  11:21:20 +0800 (0:00:00.807)       0:00:00.950 *********** 
2020-06-08 11:21:20,222 p=24602 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 11:21:21,436 p=24602 u=nisafajriyati |  changed: [default]
2020-06-08 11:21:21,448 p=24602 u=nisafajriyati |  TASK [Install aptitude] ********************************************************
2020-06-08 11:21:21,448 p=24602 u=nisafajriyati |  Monday 08 June 2020  11:21:21 +0800 (0:00:01.244)       0:00:02.195 *********** 
2020-06-08 11:21:21,470 p=24602 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 11:22:03,206 p=24602 u=nisafajriyati |   [WARNING]: Updating cache and auto-installing missing dependency: python-apt

2020-06-08 11:22:03,206 p=24602 u=nisafajriyati |   [WARNING]: Could not find aptitude. Using apt-get instead

2020-06-08 11:22:03,207 p=24602 u=nisafajriyati |  [DEPRECATION WARNING]: Distribution Ubuntu 18.04 on host default should use 
/usr/bin/python3, but is using /usr/bin/python for backward compatibility with 
prior Ansible releases. A future Ansible release will default to using the 
discovered platform python for this host. See https://docs.ansible.com/ansible/
2.8/reference_appendices/interpreter_discovery.html for more information. This 
feature will be removed in version 2.12. Deprecation warnings can be disabled 
by setting deprecation_warnings=False in ansible.cfg.
2020-06-08 11:22:03,207 p=24602 u=nisafajriyati |  changed: [default]
2020-06-08 11:22:03,214 p=24602 u=nisafajriyati |  TASK [Update all packages] *****************************************************
2020-06-08 11:22:03,215 p=24602 u=nisafajriyati |  Monday 08 June 2020  11:22:03 +0800 (0:00:41.766)       0:00:43.961 *********** 
2020-06-08 11:22:03,228 p=24602 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 11:29:35,754 p=24602 u=nisafajriyati |  fatal: [default]: FAILED! => {"changed": false, "msg": "'/usr/bin/aptitude safe-upgrade' failed: E: Failed to fetch http://archive.ubuntu.com/ubuntu/pool/main/s/systemd/udev_237-3ubuntu10.41_amd64.deb: Connection failed [IP: 91.189.88.152 80]\nE: Unable to fetch some packages; try '-o APT::Get::Fix-Missing=true' to continue with missing packages\n", "rc": 255, "stdout": "Reading package lists...\nBuilding dependency tree...\nReading state information...\nInitializing package states...\nWriting extended state information...\nBuilding tag database...\nResolving dependencies...\nThe following NEW packages will be installed:\n  libnetplan0{a} linux-headers-4.15.0-101{a} linux-headers-4.15.0-101-generic{a} linux-image-4.15.0-101-generic{a} linux-modules-4.15.0-101-generic{a} \nThe following packages will be upgraded:\n  apport apt apt-utils bind9-host ca-certificates distro-info-data dmeventd dmsetup dnsutils file git git-man iproute2 kmod libapt-inst2.0 libapt-pkg5.0 libbind9-160 libdevmapper-event1.02.1 libdevmapper1.02.1 libdns-export1100 libdns1100 libirs160 libisc-export169 libisc169 libisccc160 libisccfg160 libjson-c3 libkmod2 libldap-2.4-2 libldap-common liblvm2app2.2 liblvm2cmd2.02 liblwres160 libmagic-mgc libmagic1 libnss-systemd libpam-systemd libpython3.6 libpython3.6-minimal libpython3.6-stdlib libssl1.1 libsystemd0 libudev1 linux-headers-generic linux-headers-virtual linux-image-virtual linux-virtual lvm2 lxcfs netplan.io nplan open-iscsi open-vm-tools openssl python-apt-common python3-apport python3-apt python3-problem-report python3-software-properties python3-update-manager python3.6 python3.6-minimal software-properties-common sosreport systemd systemd-sysv tzdata udev update-manager-core \n69 packages upgraded, 5 newly installed, 0 to remove and 0 not upgraded.\nNeed to get 59.9 MB of archives. After unpacking 163 MB will be used.\nWriting extended state information...\nGet: 1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libsystemd0 amd64 237-3ubuntu10.41 [207 kB]\nGet: 2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libnss-systemd amd64 237-3ubuntu10.41 [105 kB]\nGet: 3 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpam-systemd amd64 237-3ubuntu10.41 [107 kB]\nGet: 4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 systemd amd64 237-3ubuntu10.41 [2914 kB]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 udev amd64 237-3ubuntu10.41\n  Connection failed [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libudev1 amd64 237-3ubuntu10.41\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (91.189.88.142), connection timed out Could not connect to archive.ubuntu.com:80 (91.189.88.152), connection timed out [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 kmod amd64 24-1ubuntu3.4\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libkmod2 amd64 24-1ubuntu3.4\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 systemd-sysv amd64 237-3ubuntu10.41\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libapt-pkg5.0 amd64 1.6.12ubuntu0.1\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libapt-inst2.0 amd64 1.6.12ubuntu0.1\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 apt amd64 1.6.12ubuntu0.1\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 apt-utils amd64 1.6.12ubuntu0.1\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6 amd64 3.6.9-1~18.04ubuntu1\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libssl1.1 amd64 1.1.1-1ubuntu2.1~18.04.6\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6 amd64 3.6.9-1~18.04ubuntu1\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6-stdlib amd64 3.6.9-1~18.04ubuntu1\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6-minimal amd64 3.6.9-1~18.04ubuntu1\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6-minimal amd64 3.6.9-1~18.04ubuntu1\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 open-iscsi amd64 2.0.874-5ubuntu2.10\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 openssl amd64 1.1.1-1ubuntu2.1~18.04.6\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 ca-certificates all 20190110~18.04.1\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 distro-info-data all 0.37ubuntu0.7\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdevmapper1.02.1 amd64 2:1.02.145-4.1ubuntu3.18.04.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 dmsetup amd64 2:1.02.145-4.1ubuntu3.18.04.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 file amd64 1:5.32-2ubuntu0.4\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic1 amd64 1:5.32-2ubuntu0.4\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic-mgc amd64 1:5.32-2ubuntu0.4\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 iproute2 amd64 4.15.0-2ubuntu1.1\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libisc-export169 amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdns-export1100 amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libjson-c3 amd64 0.12.1-1.3ubuntu0.3\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libnetplan0 amd64 0.99-0ubuntu3~18.04.2\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 netplan.io amd64 0.99-0ubuntu3~18.04.2\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 nplan all 0.99-0ubuntu3~18.04.2\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 tzdata all 2020a-0ubuntu0.18.04\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libirs160 amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 dnsutils amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 bind9-host amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libbind9-160 amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libisccfg160 amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libisccc160 amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdns1100 amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libisc169 amd64 1:9.11.3+dfsg-1ubuntu1.12\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 liblwres160 amd64 1:9.11.3+dfsg-1ubuntu1.12\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python-apt-common all 1.6.5ubuntu0.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-apt amd64 1.6.5ubuntu0.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-update-manager all 1:18.04.11.12\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 update-manager-core all 1:18.04.11.12\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-problem-report all 2.20.9-0ubuntu7.15\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-apport all 2.20.9-0ubuntu7.15\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 apport all 2.20.9-0ubuntu7.15\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdevmapper-event1.02.1 amd64 2:1.02.145-4.1ubuntu3.18.04.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 liblvm2cmd2.02 amd64 2.02.176-4.1ubuntu3.18.04.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 dmeventd amd64 2:1.02.145-4.1ubuntu3.18.04.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 git-man all 1:2.17.1-1ubuntu0.7\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 git amd64 1:2.17.1-1ubuntu0.7\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libldap-common all 2.4.45+dfsg-1ubuntu1.5\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libldap-2.4-2 amd64 2.4.45+dfsg-1ubuntu1.5\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 liblvm2app2.2 amd64 2.02.176-4.1ubuntu3.18.04.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-4.15.0-101 all 4.15.0-101.102\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-4.15.0-101-generic amd64 4.15.0-101.102\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-modules-4.15.0-101-generic amd64 4.15.0-101.102\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-image-4.15.0-101-generic amd64 4.15.0-101.102\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-virtual amd64 4.15.0.101.91\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-image-virtual amd64 4.15.0.101.91\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-virtual amd64 4.15.0.101.91\nIgn http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-generic amd64 4.15.0.101.91\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 lvm2 amd64 2.02.176-4.1ubuntu3.18.04.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 lxcfs amd64 3.0.3-0ubuntu1~18.04.2\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 software-properties-common all 0.96.24.32.13\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-software-properties all 0.96.24.32.13\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 sosreport amd64 3.9-1ubuntu0.18.04.3\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nErr http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 open-vm-tools amd64 2:11.0.5-4ubuntu0.18.04.1\n  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]\nGet: 5 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libapt-pkg5.0 amd64 1.6.12ubuntu0.1 [807 kB]\nGet: 6 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libapt-inst2.0 amd64 1.6.12ubuntu0.1 [54.4 kB]\nGet: 7 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 apt amd64 1.6.12ubuntu0.1 [1201 kB]\nGet: 8 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 apt-utils amd64 1.6.12ubuntu0.1 [207 kB]\nGet: 9 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6 amd64 3.6.9-1~18.04ubuntu1 [1414 kB]\nGet: 10 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libssl1.1 amd64 1.1.1-1ubuntu2.1~18.04.6 [1301 kB]\nGet: 11 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6 amd64 3.6.9-1~18.04ubuntu1 [203 kB]\nGet: 12 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6-stdlib amd64 3.6.9-1~18.04ubuntu1 [1710 kB]\nGet: 13 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6-minimal amd64 3.6.9-1~18.04ubuntu1 [1609 kB]\nGet: 14 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6-minimal amd64 3.6.9-1~18.04ubuntu1 [1609 kB]\nGet: 15 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6-minimal amd64 3.6.9-1~18.04ubuntu1 [533 kB]\nGet: 16 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 openssl amd64 1.1.1-1ubuntu2.1~18.04.6 [614 kB]\nGet: 17 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 ca-certificates all 20190110~18.04.1 [146 kB]\nGet: 18 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 ca-certificates all 20190110~18.04.1 [146 kB]\nGet: 19 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 file amd64 1:5.32-2ubuntu0.4 [22.1 kB]\nGet: 20 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic1 amd64 1:5.32-2ubuntu0.4 [68.6 kB]\nGet: 21 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic-mgc amd64 1:5.32-2ubuntu0.4 [184 kB]\nGet: 22 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 iproute2 amd64 4.15.0-2ubuntu1.1 [723 kB]\nGet: 23 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libisc-export169 amd64 1:9.11.3+dfsg-1ubuntu1.12 [163 kB]\nGet: 24 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libdns-export1100 amd64 1:9.11.3+dfsg-1ubuntu1.12 [748 kB]\nGet: 25 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libjson-c3 amd64 0.12.1-1.3ubuntu0.3 [21.7 kB]\nGet: 26 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 tzdata all 2020a-0ubuntu0.18.04 [190 kB]\nGet: 27 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 tzdata all 2020a-0ubuntu0.18.04 [190 kB]\nGet: 28 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libirs160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [19.1 kB]\nGet: 29 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 dnsutils amd64 1:9.11.3+dfsg-1ubuntu1.12 [145 kB]\nGet: 30 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 bind9-host amd64 1:9.11.3+dfsg-1ubuntu1.12 [53.4 kB]\nGet: 31 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libbind9-160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [27.6 kB]\nGet: 32 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libisccfg160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [48.4 kB]\nGet: 33 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libisccc160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [17.9 kB]\nGet: 34 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libdns1100 amd64 1:9.11.3+dfsg-1ubuntu1.12 [967 kB]\nGet: 35 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libisc169 amd64 1:9.11.3+dfsg-1ubuntu1.12 [238 kB]\nGet: 36 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 liblwres160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [34.6 kB]\nGet: 37 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 git-man all 1:2.17.1-1ubuntu0.7 [804 kB]\nGet: 38 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 git amd64 1:2.17.1-1ubuntu0.7 [3915 kB]\nGet: 39 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libldap-common all 2.4.45+dfsg-1ubuntu1.5 [16.9 kB]\nGet: 40 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libldap-2.4-2 amd64 2.4.45+dfsg-1ubuntu1.5 [155 kB]\nGet: 41 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-4.15.0-101 all 4.15.0-101.102 [10.9 MB]\nGet: 42 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-4.15.0-101-generic amd64 4.15.0-101.102 [1100 kB]\nGet: 43 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-modules-4.15.0-101-generic amd64 4.15.0-101.102 [13.0 MB]\nGet: 44 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-image-4.15.0-101-generic amd64 4.15.0-101.102 [8005 kB]\nGet: 45 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-virtual amd64 4.15.0.101.91 [1860 B]\nGet: 46 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-image-virtual amd64 4.15.0.101.91 [2444 B]\nGet: 47 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-virtual amd64 4.15.0.101.91 [1840 B]\nGet: 48 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-generic amd64 4.15.0.101.91 [2424 B]\nFetched 54.4 MB in 7min 27s (122 kB/s)\nReading package lists...\nBuilding dependency tree...\nReading state information...\nReading extended state information...\nInitializing package states...\nWriting extended state information...\nBuilding tag database...\n", "stdout_lines": ["Reading package lists...", "Building dependency tree...", "Reading state information...", "Initializing package states...", "Writing extended state information...", "Building tag database...", "Resolving dependencies...", "The following NEW packages will be installed:", "  libnetplan0{a} linux-headers-4.15.0-101{a} linux-headers-4.15.0-101-generic{a} linux-image-4.15.0-101-generic{a} linux-modules-4.15.0-101-generic{a} ", "The following packages will be upgraded:", "  apport apt apt-utils bind9-host ca-certificates distro-info-data dmeventd dmsetup dnsutils file git git-man iproute2 kmod libapt-inst2.0 libapt-pkg5.0 libbind9-160 libdevmapper-event1.02.1 libdevmapper1.02.1 libdns-export1100 libdns1100 libirs160 libisc-export169 libisc169 libisccc160 libisccfg160 libjson-c3 libkmod2 libldap-2.4-2 libldap-common liblvm2app2.2 liblvm2cmd2.02 liblwres160 libmagic-mgc libmagic1 libnss-systemd libpam-systemd libpython3.6 libpython3.6-minimal libpython3.6-stdlib libssl1.1 libsystemd0 libudev1 linux-headers-generic linux-headers-virtual linux-image-virtual linux-virtual lvm2 lxcfs netplan.io nplan open-iscsi open-vm-tools openssl python-apt-common python3-apport python3-apt python3-problem-report python3-software-properties python3-update-manager python3.6 python3.6-minimal software-properties-common sosreport systemd systemd-sysv tzdata udev update-manager-core ", "69 packages upgraded, 5 newly installed, 0 to remove and 0 not upgraded.", "Need to get 59.9 MB of archives. After unpacking 163 MB will be used.", "Writing extended state information...", "Get: 1 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libsystemd0 amd64 237-3ubuntu10.41 [207 kB]", "Get: 2 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libnss-systemd amd64 237-3ubuntu10.41 [105 kB]", "Get: 3 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpam-systemd amd64 237-3ubuntu10.41 [107 kB]", "Get: 4 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 systemd amd64 237-3ubuntu10.41 [2914 kB]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 udev amd64 237-3ubuntu10.41", "  Connection failed [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libudev1 amd64 237-3ubuntu10.41", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) Could not connect to archive.ubuntu.com:80 (91.189.88.142), connection timed out Could not connect to archive.ubuntu.com:80 (91.189.88.152), connection timed out [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 kmod amd64 24-1ubuntu3.4", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libkmod2 amd64 24-1ubuntu3.4", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 systemd-sysv amd64 237-3ubuntu10.41", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libapt-pkg5.0 amd64 1.6.12ubuntu0.1", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libapt-inst2.0 amd64 1.6.12ubuntu0.1", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 apt amd64 1.6.12ubuntu0.1", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 apt-utils amd64 1.6.12ubuntu0.1", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6 amd64 3.6.9-1~18.04ubuntu1", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libssl1.1 amd64 1.1.1-1ubuntu2.1~18.04.6", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6 amd64 3.6.9-1~18.04ubuntu1", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6-stdlib amd64 3.6.9-1~18.04ubuntu1", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6-minimal amd64 3.6.9-1~18.04ubuntu1", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6-minimal amd64 3.6.9-1~18.04ubuntu1", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 open-iscsi amd64 2.0.874-5ubuntu2.10", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 openssl amd64 1.1.1-1ubuntu2.1~18.04.6", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 ca-certificates all 20190110~18.04.1", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 distro-info-data all 0.37ubuntu0.7", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdevmapper1.02.1 amd64 2:1.02.145-4.1ubuntu3.18.04.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 dmsetup amd64 2:1.02.145-4.1ubuntu3.18.04.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 file amd64 1:5.32-2ubuntu0.4", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic1 amd64 1:5.32-2ubuntu0.4", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic-mgc amd64 1:5.32-2ubuntu0.4", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 iproute2 amd64 4.15.0-2ubuntu1.1", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libisc-export169 amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdns-export1100 amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libjson-c3 amd64 0.12.1-1.3ubuntu0.3", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libnetplan0 amd64 0.99-0ubuntu3~18.04.2", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 netplan.io amd64 0.99-0ubuntu3~18.04.2", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 nplan all 0.99-0ubuntu3~18.04.2", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 tzdata all 2020a-0ubuntu0.18.04", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libirs160 amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 dnsutils amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 bind9-host amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libbind9-160 amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libisccfg160 amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libisccc160 amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdns1100 amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libisc169 amd64 1:9.11.3+dfsg-1ubuntu1.12", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 liblwres160 amd64 1:9.11.3+dfsg-1ubuntu1.12", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python-apt-common all 1.6.5ubuntu0.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-apt amd64 1.6.5ubuntu0.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-update-manager all 1:18.04.11.12", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 update-manager-core all 1:18.04.11.12", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-problem-report all 2.20.9-0ubuntu7.15", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-apport all 2.20.9-0ubuntu7.15", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 apport all 2.20.9-0ubuntu7.15", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libdevmapper-event1.02.1 amd64 2:1.02.145-4.1ubuntu3.18.04.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 liblvm2cmd2.02 amd64 2.02.176-4.1ubuntu3.18.04.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 dmeventd amd64 2:1.02.145-4.1ubuntu3.18.04.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 git-man all 1:2.17.1-1ubuntu0.7", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 git amd64 1:2.17.1-1ubuntu0.7", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libldap-common all 2.4.45+dfsg-1ubuntu1.5", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libldap-2.4-2 amd64 2.4.45+dfsg-1ubuntu1.5", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 liblvm2app2.2 amd64 2.02.176-4.1ubuntu3.18.04.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-4.15.0-101 all 4.15.0-101.102", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-4.15.0-101-generic amd64 4.15.0-101.102", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-modules-4.15.0-101-generic amd64 4.15.0-101.102", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-image-4.15.0-101-generic amd64 4.15.0-101.102", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-virtual amd64 4.15.0.101.91", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-image-virtual amd64 4.15.0.101.91", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-virtual amd64 4.15.0.101.91", "Ign http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-generic amd64 4.15.0.101.91", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 lvm2 amd64 2.02.176-4.1ubuntu3.18.04.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 lxcfs amd64 3.0.3-0ubuntu1~18.04.2", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 software-properties-common all 0.96.24.32.13", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python3-software-properties all 0.96.24.32.13", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 sosreport amd64 3.9-1ubuntu0.18.04.3", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Err http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 open-vm-tools amd64 2:11.0.5-4ubuntu0.18.04.1", "  Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::18). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::24). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1562::15). - connect (101: Network is unreachable) Cannot initiate the connection to archive.ubuntu.com:80 (2001:67c:1360:8001::23). - connect (101: Network is unreachable) [IP: 91.189.88.152 80]", "Get: 5 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libapt-pkg5.0 amd64 1.6.12ubuntu0.1 [807 kB]", "Get: 6 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libapt-inst2.0 amd64 1.6.12ubuntu0.1 [54.4 kB]", "Get: 7 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 apt amd64 1.6.12ubuntu0.1 [1201 kB]", "Get: 8 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 apt-utils amd64 1.6.12ubuntu0.1 [207 kB]", "Get: 9 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6 amd64 3.6.9-1~18.04ubuntu1 [1414 kB]", "Get: 10 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libssl1.1 amd64 1.1.1-1ubuntu2.1~18.04.6 [1301 kB]", "Get: 11 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6 amd64 3.6.9-1~18.04ubuntu1 [203 kB]", "Get: 12 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6-stdlib amd64 3.6.9-1~18.04ubuntu1 [1710 kB]", "Get: 13 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6-minimal amd64 3.6.9-1~18.04ubuntu1 [1609 kB]", "Get: 14 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 python3.6-minimal amd64 3.6.9-1~18.04ubuntu1 [1609 kB]", "Get: 15 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libpython3.6-minimal amd64 3.6.9-1~18.04ubuntu1 [533 kB]", "Get: 16 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 openssl amd64 1.1.1-1ubuntu2.1~18.04.6 [614 kB]", "Get: 17 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 ca-certificates all 20190110~18.04.1 [146 kB]", "Get: 18 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 ca-certificates all 20190110~18.04.1 [146 kB]", "Get: 19 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 file amd64 1:5.32-2ubuntu0.4 [22.1 kB]", "Get: 20 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic1 amd64 1:5.32-2ubuntu0.4 [68.6 kB]", "Get: 21 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libmagic-mgc amd64 1:5.32-2ubuntu0.4 [184 kB]", "Get: 22 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 iproute2 amd64 4.15.0-2ubuntu1.1 [723 kB]", "Get: 23 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libisc-export169 amd64 1:9.11.3+dfsg-1ubuntu1.12 [163 kB]", "Get: 24 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libdns-export1100 amd64 1:9.11.3+dfsg-1ubuntu1.12 [748 kB]", "Get: 25 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libjson-c3 amd64 0.12.1-1.3ubuntu0.3 [21.7 kB]", "Get: 26 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 tzdata all 2020a-0ubuntu0.18.04 [190 kB]", "Get: 27 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 tzdata all 2020a-0ubuntu0.18.04 [190 kB]", "Get: 28 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libirs160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [19.1 kB]", "Get: 29 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 dnsutils amd64 1:9.11.3+dfsg-1ubuntu1.12 [145 kB]", "Get: 30 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 bind9-host amd64 1:9.11.3+dfsg-1ubuntu1.12 [53.4 kB]", "Get: 31 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libbind9-160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [27.6 kB]", "Get: 32 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libisccfg160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [48.4 kB]", "Get: 33 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libisccc160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [17.9 kB]", "Get: 34 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libdns1100 amd64 1:9.11.3+dfsg-1ubuntu1.12 [967 kB]", "Get: 35 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libisc169 amd64 1:9.11.3+dfsg-1ubuntu1.12 [238 kB]", "Get: 36 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 liblwres160 amd64 1:9.11.3+dfsg-1ubuntu1.12 [34.6 kB]", "Get: 37 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 git-man all 1:2.17.1-1ubuntu0.7 [804 kB]", "Get: 38 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 git amd64 1:2.17.1-1ubuntu0.7 [3915 kB]", "Get: 39 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libldap-common all 2.4.45+dfsg-1ubuntu1.5 [16.9 kB]", "Get: 40 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 libldap-2.4-2 amd64 2.4.45+dfsg-1ubuntu1.5 [155 kB]", "Get: 41 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-4.15.0-101 all 4.15.0-101.102 [10.9 MB]", "Get: 42 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-4.15.0-101-generic amd64 4.15.0-101.102 [1100 kB]", "Get: 43 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-modules-4.15.0-101-generic amd64 4.15.0-101.102 [13.0 MB]", "Get: 44 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-image-4.15.0-101-generic amd64 4.15.0-101.102 [8005 kB]", "Get: 45 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-virtual amd64 4.15.0.101.91 [1860 B]", "Get: 46 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-image-virtual amd64 4.15.0.101.91 [2444 B]", "Get: 47 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-virtual amd64 4.15.0.101.91 [1840 B]", "Get: 48 http://security.ubuntu.com/ubuntu bionic-updates/main amd64 linux-headers-generic amd64 4.15.0.101.91 [2424 B]", "Fetched 54.4 MB in 7min 27s (122 kB/s)", "Reading package lists...", "Building dependency tree...", "Reading state information...", "Reading extended state information...", "Initializing package states...", "Writing extended state information...", "Building tag database..."]}
2020-06-08 11:29:35,758 p=24602 u=nisafajriyati |  PLAY RECAP *********************************************************************
2020-06-08 11:29:35,758 p=24602 u=nisafajriyati |  default                    : ok=3    changed=2    unreachable=0    failed=1    skipped=0    rescued=0    ignored=1   
2020-06-08 11:29:35,758 p=24602 u=nisafajriyati |  Monday 08 June 2020  11:29:35 +0800 (0:07:32.543)       0:08:16.505 *********** 
2020-06-08 11:29:35,758 p=24602 u=nisafajriyati |  =============================================================================== 
2020-06-08 11:29:35,759 p=24602 u=nisafajriyati |  Update all packages --------------------------------------------------- 452.54s
2020-06-08 11:29:35,759 p=24602 u=nisafajriyati |  Install aptitude ------------------------------------------------------- 41.77s
2020-06-08 11:29:35,759 p=24602 u=nisafajriyati |  Install python2 --------------------------------------------------------- 1.24s
2020-06-08 11:29:35,759 p=24602 u=nisafajriyati |  Check if user specified in config is installed -------------------------- 0.81s
2020-06-08 13:07:29,391 p=1555 u=vagrant n=ansible | - downloading role 'swapfile', owned by kamaln7
2020-06-08 13:08:01,334 p=1555 u=vagrant n=ansible | [WARNING]: Unable to retrieve role (id=1262) data (versions), but this is not
fatal so we continue: Unknown error when attempting to call Galaxy at
'https://galaxy.ansible.com/api/v1/roles/1262/versions/?page_size=50': <urlopen
error ('_ssl.c:711: The handshake operation timed out',)>

2020-06-08 13:08:01,334 p=1555 u=vagrant n=ansible | - downloading role from https://github.com/kamaln7/ansible-swapfile/archive/master.tar.gz
2020-06-08 13:08:03,060 p=1555 u=vagrant n=ansible | - extracting kamaln7.swapfile to /etc/ansible/roles/kamaln7.swapfile
2020-06-08 13:08:03,061 p=1555 u=vagrant n=ansible | - extracting kamaln7.swapfile to /home/vagrant/.ansible/roles/kamaln7.swapfile
2020-06-08 13:08:03,066 p=1555 u=vagrant n=ansible | - kamaln7.swapfile (master) was installed successfully
2020-06-08 13:08:03,069 p=1555 u=vagrant n=ansible | - downloading role 'nodejs', owned by geerlingguy
2020-06-08 13:08:20,193 p=1555 u=vagrant n=ansible | - downloading role from https://github.com/geerlingguy/ansible-role-nodejs/archive/5.1.1.tar.gz
2020-06-08 13:08:22,386 p=1555 u=vagrant n=ansible | - extracting geerlingguy.nodejs to /etc/ansible/roles/geerlingguy.nodejs
2020-06-08 13:08:22,387 p=1555 u=vagrant n=ansible | - extracting geerlingguy.nodejs to /home/vagrant/.ansible/roles/geerlingguy.nodejs
2020-06-08 13:08:22,393 p=1555 u=vagrant n=ansible | - geerlingguy.nodejs (5.1.1) was installed successfully
2020-06-08 13:08:22,396 p=1555 u=vagrant n=ansible | - downloading role 'composer', owned by geerlingguy
2020-06-08 13:08:31,558 p=1555 u=vagrant n=ansible | - downloading role from https://github.com/geerlingguy/ansible-role-composer/archive/1.7.6.tar.gz
2020-06-08 13:08:33,021 p=1555 u=vagrant n=ansible | - extracting geerlingguy.composer to /etc/ansible/roles/geerlingguy.composer
2020-06-08 13:08:33,021 p=1555 u=vagrant n=ansible | - extracting geerlingguy.composer to /home/vagrant/.ansible/roles/geerlingguy.composer
2020-06-08 13:08:33,025 p=1555 u=vagrant n=ansible | - geerlingguy.composer (1.7.6) was installed successfully
2020-06-08 13:08:33,027 p=1555 u=vagrant n=ansible | - downloading role 'mysql', owned by geerlingguy
2020-06-08 13:08:52,640 p=1555 u=vagrant n=ansible | - downloading role from https://github.com/geerlingguy/ansible-role-mysql/archive/3.1.0.tar.gz
2020-06-08 13:08:53,980 p=1555 u=vagrant n=ansible | - extracting geerlingguy.mysql to /etc/ansible/roles/geerlingguy.mysql
2020-06-08 13:08:53,982 p=1555 u=vagrant n=ansible | - extracting geerlingguy.mysql to /home/vagrant/.ansible/roles/geerlingguy.mysql
2020-06-08 13:08:53,985 p=1555 u=vagrant n=ansible | - geerlingguy.mysql (3.1.0) was installed successfully
2020-06-08 13:08:54,965 p=1607 u=vagrant n=ansible | ERROR! the role 'kamaln7.swapfile' was not found in /vagrant/roles:/etc/ansible/roles:/vagrant

The error appears to be in '/vagrant/setup.yml': line 22, column 7, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:


    - name: kamaln7.swapfile
      ^ here

2020-06-08 13:12:39,229 p=1861 u=vagrant n=ansible | - downloading role 'swapfile', owned by oefenweb
2020-06-08 13:12:44,954 p=1861 u=vagrant n=ansible | - downloading role from https://github.com/Oefenweb/ansible-swapfile/archive/v2.0.26.tar.gz
2020-06-08 13:12:46,913 p=1861 u=vagrant n=ansible | - extracting oefenweb.swapfile to /etc/ansible/roles/oefenweb.swapfile
2020-06-08 13:12:46,914 p=1861 u=vagrant n=ansible | - extracting oefenweb.swapfile to /home/vagrant/.ansible/roles/oefenweb.swapfile
2020-06-08 13:12:46,924 p=1861 u=vagrant n=ansible | - oefenweb.swapfile (v2.0.26) was installed successfully
2020-06-08 13:12:46,942 p=1861 u=vagrant n=ansible | - downloading role 'nodejs', owned by geerlingguy
2020-06-08 13:13:09,849 p=1861 u=vagrant n=ansible | ERROR! Unexpected Exception, this is probably a bug: ('The read operation timed out',)
2020-06-08 13:13:09,850 p=1861 u=vagrant n=ansible | to see the full traceback, use -vvv
2020-06-08 13:13:09,861 p=1861 u=vagrant n=ansible | the full traceback was:

Traceback (most recent call last):
  File "/usr/bin/ansible-galaxy", line 123, in <module>
    exit_code = cli.run()
  File "/usr/lib/python2.7/dist-packages/ansible/cli/galaxy.py", line 375, in run
    context.CLIARGS['func']()
  File "/usr/lib/python2.7/dist-packages/ansible/cli/galaxy.py", line 891, in execute_install
    installed = role.install()
  File "/usr/lib/python2.7/dist-packages/ansible/galaxy/role.py", line 218, in install
    role_data = self.api.lookup_role_by_name(self.src)
  File "/usr/lib/python2.7/dist-packages/ansible/galaxy/api.py", line 98, in wrapped
    return method(self, *args, **kwargs)
  File "/usr/lib/python2.7/dist-packages/ansible/galaxy/api.py", line 289, in lookup_role_by_name
    data = self._call_galaxy(url)
  File "/usr/lib/python2.7/dist-packages/ansible/galaxy/api.py", line 203, in _call_galaxy
    resp_data = to_text(resp.read(), errors='surrogate_or_strict')
  File "/usr/lib/python2.7/socket.py", line 355, in read
    data = self._sock.recv(rbufsize)
  File "/usr/lib/python2.7/httplib.py", line 607, in read
    s = self.fp.read(amt)
  File "/usr/lib/python2.7/socket.py", line 384, in read
    data = self._sock.recv(left)
  File "/usr/lib/python2.7/ssl.py", line 754, in recv
    return self.read(buflen)
  File "/usr/lib/python2.7/ssl.py", line 641, in read
    v = self._sslobj.read(len)
SSLError: ('The read operation timed out',)

2020-06-08 13:14:13,545 p=2115 u=vagrant n=ansible | - downloading role 'swapfile', owned by oefenweb
2020-06-08 13:14:18,011 p=2115 u=vagrant n=ansible | - downloading role from https://github.com/Oefenweb/ansible-swapfile/archive/v2.0.26.tar.gz
2020-06-08 13:14:19,159 p=2115 u=vagrant n=ansible | - extracting oefenweb.swapfile to /etc/ansible/roles/oefenweb.swapfile
2020-06-08 13:14:19,161 p=2115 u=vagrant n=ansible | - extracting oefenweb.swapfile to /home/vagrant/.ansible/roles/oefenweb.swapfile
2020-06-08 13:14:19,165 p=2115 u=vagrant n=ansible | [WARNING]: - oefenweb.swapfile was NOT installed successfully: the specified
role oefenweb.swapfile appears to already exist. Use --force to replace it.

2020-06-08 13:14:19,167 p=2115 u=vagrant n=ansible | ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.
2020-06-08 21:15:14,901 p=67347 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-06-08 21:15:19,515 p=67347 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-06-08 21:15:19,538 p=67347 u=nisafajriyati |  TASK [Check if user specified in config is installed] **************************
2020-06-08 21:15:19,539 p=67347 u=nisafajriyati |  Monday 08 June 2020  21:15:19 +0800 (0:00:00.137)       0:00:00.137 *********** 
2020-06-08 21:15:20,214 p=67347 u=nisafajriyati |  fatal: [default -> localhost]: FAILED! => {"changed": false, "cmd": ["ssh", "-q", "-o", "BatchMode=yes", "-o", "ConnectTimeout=3", "default", "echo ok"], "delta": "0:00:00.259560", "end": "2020-06-08 21:15:20.184211", "msg": "non-zero return code", "rc": 255, "start": "2020-06-08 21:15:19.924651", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
2020-06-08 21:15:20,214 p=67347 u=nisafajriyati |  ...ignoring
2020-06-08 21:15:20,223 p=67347 u=nisafajriyati |  TASK [Install python2] *********************************************************
2020-06-08 21:15:20,223 p=67347 u=nisafajriyati |  Monday 08 June 2020  21:15:20 +0800 (0:00:00.684)       0:00:00.821 *********** 
2020-06-08 21:15:20,241 p=67347 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:15:21,387 p=67347 u=nisafajriyati |  changed: [default]
2020-06-08 21:15:21,394 p=67347 u=nisafajriyati |  TASK [Install aptitude] ********************************************************
2020-06-08 21:15:21,395 p=67347 u=nisafajriyati |  Monday 08 June 2020  21:15:21 +0800 (0:00:01.171)       0:00:01.993 *********** 
2020-06-08 21:15:21,408 p=67347 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:15:22,774 p=67347 u=nisafajriyati |  [DEPRECATION WARNING]: Distribution Ubuntu 18.04 on host default should use 
/usr/bin/python3, but is using /usr/bin/python for backward compatibility with 
prior Ansible releases. A future Ansible release will default to using the 
discovered platform python for this host. See https://docs.ansible.com/ansible/
2.8/reference_appendices/interpreter_discovery.html for more information. This 
feature will be removed in version 2.12. Deprecation warnings can be disabled 
by setting deprecation_warnings=False in ansible.cfg.
2020-06-08 21:15:22,774 p=67347 u=nisafajriyati |  ok: [default]
2020-06-08 21:15:22,781 p=67347 u=nisafajriyati |  TASK [Update all packages] *****************************************************
2020-06-08 21:15:22,782 p=67347 u=nisafajriyati |  Monday 08 June 2020  21:15:22 +0800 (0:00:01.386)       0:00:03.380 *********** 
2020-06-08 21:15:22,808 p=67347 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:17:47,323 p=67347 u=nisafajriyati |  changed: [default]
2020-06-08 21:17:47,338 p=67347 u=nisafajriyati |  TASK [user : Ensure sudo group is present] *************************************
2020-06-08 21:17:47,338 p=67347 u=nisafajriyati |  Monday 08 June 2020  21:17:47 +0800 (0:02:24.556)       0:02:27.936 *********** 
2020-06-08 21:17:47,353 p=67347 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:17:47,823 p=67347 u=nisafajriyati |  ok: [default]
2020-06-08 21:17:47,831 p=67347 u=nisafajriyati |  TASK [user : Ensure sudo group has sudo privileges] ****************************
2020-06-08 21:17:47,832 p=67347 u=nisafajriyati |  Monday 08 June 2020  21:17:47 +0800 (0:00:00.493)       0:02:28.430 *********** 
2020-06-08 21:17:47,854 p=67347 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:17:48,211 p=67347 u=nisafajriyati |  changed: [default]
2020-06-08 21:17:48,218 p=67347 u=nisafajriyati |  TASK [user : Add new super user] ***********************************************
2020-06-08 21:17:48,218 p=67347 u=nisafajriyati |  Monday 08 June 2020  21:17:48 +0800 (0:00:00.386)       0:02:28.816 *********** 
2020-06-08 21:17:48,240 p=67347 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:17:48,752 p=67347 u=nisafajriyati |  changed: [default]
2020-06-08 21:17:48,761 p=67347 u=nisafajriyati |  TASK [user : Add ssh key for new super user] ***********************************
2020-06-08 21:17:48,762 p=67347 u=nisafajriyati |  Monday 08 June 2020  21:17:48 +0800 (0:00:00.543)       0:02:29.360 *********** 
2020-06-08 21:17:48,794 p=67347 u=nisafajriyati |   [WARNING]: Unable to find '~/.ssh/authorized_keys' in expected paths (use
-vvvvv to see paths)

2020-06-08 21:17:48,798 p=67347 u=nisafajriyati |  fatal: [default]: FAILED! => {"msg": "An unhandled exception occurred while running the lookup plugin 'file'. Error was a <class 'ansible.errors.AnsibleError'>, original message: could not locate file in lookup: ~/.ssh/authorized_keys"}
2020-06-08 21:17:48,803 p=67347 u=nisafajriyati |  PLAY RECAP *********************************************************************
2020-06-08 21:17:48,804 p=67347 u=nisafajriyati |  default                    : ok=7    changed=4    unreachable=0    failed=1    skipped=0    rescued=0    ignored=1   
2020-06-08 21:17:48,804 p=67347 u=nisafajriyati |  Monday 08 June 2020  21:17:48 +0800 (0:00:00.042)       0:02:29.402 *********** 
2020-06-08 21:17:48,804 p=67347 u=nisafajriyati |  =============================================================================== 
2020-06-08 21:17:48,804 p=67347 u=nisafajriyati |  Update all packages --------------------------------------------------- 144.56s
2020-06-08 21:17:48,804 p=67347 u=nisafajriyati |  Install aptitude -------------------------------------------------------- 1.39s
2020-06-08 21:17:48,805 p=67347 u=nisafajriyati |  Install python2 --------------------------------------------------------- 1.17s
2020-06-08 21:17:48,805 p=67347 u=nisafajriyati |  Check if user specified in config is installed -------------------------- 0.68s
2020-06-08 21:17:48,805 p=67347 u=nisafajriyati |  user : Add new super user ----------------------------------------------- 0.54s
2020-06-08 21:17:48,805 p=67347 u=nisafajriyati |  user : Ensure sudo group is present ------------------------------------- 0.49s
2020-06-08 21:17:48,805 p=67347 u=nisafajriyati |  user : Ensure sudo group has sudo privileges ---------------------------- 0.39s
2020-06-08 21:17:48,805 p=67347 u=nisafajriyati |  user : Add ssh key for new super user ----------------------------------- 0.04s
2020-06-08 21:19:51,464 p=68076 u=nisafajriyati |  [DEPRECATION WARNING]: DEFAULT_ASK_SUDO_PASS option, In favor of Ansible 
Become, which is a generic framework. See become_ask_pass. , use become 
instead. This feature will be removed in version 2.9. Deprecation warnings can 
be disabled by setting deprecation_warnings=False in ansible.cfg.
2020-06-08 21:19:56,991 p=68076 u=nisafajriyati |  PLAY [all] *********************************************************************
2020-06-08 21:19:57,019 p=68076 u=nisafajriyati |  TASK [Check if user specified in config is installed] **************************
2020-06-08 21:19:57,019 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:19:57 +0800 (0:00:00.140)       0:00:00.140 *********** 
2020-06-08 21:19:57,458 p=68076 u=nisafajriyati |  fatal: [default -> localhost]: FAILED! => {"changed": false, "cmd": ["ssh", "-q", "-o", "BatchMode=yes", "-o", "ConnectTimeout=3", "default", "echo ok"], "delta": "0:00:00.016052", "end": "2020-06-08 21:19:57.428002", "msg": "non-zero return code", "rc": 255, "start": "2020-06-08 21:19:57.411950", "stderr": "", "stderr_lines": [], "stdout": "", "stdout_lines": []}
2020-06-08 21:19:57,459 p=68076 u=nisafajriyati |  ...ignoring
2020-06-08 21:19:57,465 p=68076 u=nisafajriyati |  TASK [Install python2] *********************************************************
2020-06-08 21:19:57,465 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:19:57 +0800 (0:00:00.445)       0:00:00.586 *********** 
2020-06-08 21:19:57,481 p=68076 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:19:59,165 p=68076 u=nisafajriyati |  changed: [default]
2020-06-08 21:19:59,172 p=68076 u=nisafajriyati |  TASK [Install aptitude] ********************************************************
2020-06-08 21:19:59,172 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:19:59 +0800 (0:00:01.707)       0:00:02.293 *********** 
2020-06-08 21:19:59,187 p=68076 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:20:00,206 p=68076 u=nisafajriyati |  [DEPRECATION WARNING]: Distribution Ubuntu 18.04 on host default should use 
/usr/bin/python3, but is using /usr/bin/python for backward compatibility with 
prior Ansible releases. A future Ansible release will default to using the 
discovered platform python for this host. See https://docs.ansible.com/ansible/
2.8/reference_appendices/interpreter_discovery.html for more information. This 
feature will be removed in version 2.12. Deprecation warnings can be disabled 
by setting deprecation_warnings=False in ansible.cfg.
2020-06-08 21:20:00,207 p=68076 u=nisafajriyati |  ok: [default]
2020-06-08 21:20:00,214 p=68076 u=nisafajriyati |  TASK [Update all packages] *****************************************************
2020-06-08 21:20:00,214 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:00 +0800 (0:00:01.042)       0:00:03.336 *********** 
2020-06-08 21:20:00,233 p=68076 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:20:06,721 p=68076 u=nisafajriyati |  ok: [default]
2020-06-08 21:20:06,728 p=68076 u=nisafajriyati |  TASK [user : Ensure sudo group is present] *************************************
2020-06-08 21:20:06,728 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:06 +0800 (0:00:06.514)       0:00:09.850 *********** 
2020-06-08 21:20:06,749 p=68076 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:20:07,135 p=68076 u=nisafajriyati |  ok: [default]
2020-06-08 21:20:07,144 p=68076 u=nisafajriyati |  TASK [user : Ensure sudo group has sudo privileges] ****************************
2020-06-08 21:20:07,144 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:07 +0800 (0:00:00.415)       0:00:10.265 *********** 
2020-06-08 21:20:07,159 p=68076 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:20:07,489 p=68076 u=nisafajriyati |  ok: [default]
2020-06-08 21:20:07,497 p=68076 u=nisafajriyati |  TASK [user : Add new super user] ***********************************************
2020-06-08 21:20:07,498 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:07 +0800 (0:00:00.353)       0:00:10.619 *********** 
2020-06-08 21:20:07,514 p=68076 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:20:07,916 p=68076 u=nisafajriyati |  ok: [default]
2020-06-08 21:20:07,924 p=68076 u=nisafajriyati |  TASK [user : Add ssh key for new super user] ***********************************
2020-06-08 21:20:07,924 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:07 +0800 (0:00:00.426)       0:00:11.045 *********** 
2020-06-08 21:20:07,960 p=68076 u=nisafajriyati |  [DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using 
`result|success` use `result is success`. This feature will be removed in 
version 2.9. Deprecation warnings can be disabled by setting 
deprecation_warnings=False in ansible.cfg.
2020-06-08 21:20:08,355 p=68076 u=nisafajriyati |  changed: [default]
2020-06-08 21:20:08,363 p=68076 u=nisafajriyati |  TASK [ssh : Ensure ssh is installed] *******************************************
2020-06-08 21:20:08,363 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:08 +0800 (0:00:00.438)       0:00:11.484 *********** 
2020-06-08 21:20:09,115 p=68076 u=nisafajriyati |  ok: [default]
2020-06-08 21:20:09,124 p=68076 u=nisafajriyati |  TASK [ssh : Disable root login] ************************************************
2020-06-08 21:20:09,124 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:09 +0800 (0:00:00.761)       0:00:12.245 *********** 
2020-06-08 21:20:09,325 p=68076 u=nisafajriyati |  changed: [default]
2020-06-08 21:20:09,331 p=68076 u=nisafajriyati |  TASK [ssh : Disable password authentication] ***********************************
2020-06-08 21:20:09,332 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:09 +0800 (0:00:00.207)       0:00:12.453 *********** 
2020-06-08 21:20:09,527 p=68076 u=nisafajriyati |  ok: [default]
2020-06-08 21:20:09,535 p=68076 u=nisafajriyati |  TASK [ssh : Only allow ssh access for the newly created user] ******************
2020-06-08 21:20:09,535 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:09 +0800 (0:00:00.203)       0:00:12.656 *********** 
2020-06-08 21:20:09,740 p=68076 u=nisafajriyati |  changed: [default]
2020-06-08 21:20:09,740 p=68076 u=nisafajriyati |  RUNNING HANDLER [ssh : Restart ssh] ********************************************
2020-06-08 21:20:09,741 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:09 +0800 (0:00:00.205)       0:00:12.862 *********** 
2020-06-08 21:20:11,212 p=68076 u=nisafajriyati |  changed: [default]
2020-06-08 21:20:11,215 p=68076 u=nisafajriyati |  PLAY RECAP *********************************************************************
2020-06-08 21:20:11,216 p=68076 u=nisafajriyati |  default                    : ok=13   changed=5    unreachable=0    failed=0    skipped=0    rescued=0    ignored=1   
2020-06-08 21:20:11,216 p=68076 u=nisafajriyati |  Monday 08 June 2020  21:20:11 +0800 (0:00:01.475)       0:00:14.337 *********** 
2020-06-08 21:20:11,216 p=68076 u=nisafajriyati |  =============================================================================== 
2020-06-08 21:20:11,216 p=68076 u=nisafajriyati |  Update all packages ----------------------------------------------------- 6.51s
2020-06-08 21:20:11,217 p=68076 u=nisafajriyati |  Install python2 --------------------------------------------------------- 1.71s
2020-06-08 21:20:11,217 p=68076 u=nisafajriyati |  ssh : Restart ssh ------------------------------------------------------- 1.48s
2020-06-08 21:20:11,217 p=68076 u=nisafajriyati |  Install aptitude -------------------------------------------------------- 1.04s
2020-06-08 21:20:11,217 p=68076 u=nisafajriyati |  ssh : Ensure ssh is installed ------------------------------------------- 0.76s
2020-06-08 21:20:11,217 p=68076 u=nisafajriyati |  Check if user specified in config is installed -------------------------- 0.45s
2020-06-08 21:20:11,217 p=68076 u=nisafajriyati |  user : Add ssh key for new super user ----------------------------------- 0.44s
2020-06-08 21:20:11,217 p=68076 u=nisafajriyati |  user : Add new super user ----------------------------------------------- 0.43s
2020-06-08 21:20:11,217 p=68076 u=nisafajriyati |  user : Ensure sudo group is present ------------------------------------- 0.42s
2020-06-08 21:20:11,218 p=68076 u=nisafajriyati |  user : Ensure sudo group has sudo privileges ---------------------------- 0.35s
2020-06-08 21:20:11,218 p=68076 u=nisafajriyati |  ssh : Disable root login ------------------------------------------------ 0.21s
2020-06-08 21:20:11,218 p=68076 u=nisafajriyati |  ssh : Only allow ssh access for the newly created user ------------------ 0.21s
2020-06-08 21:20:11,218 p=68076 u=nisafajriyati |  ssh : Disable password authentication ----------------------------------- 0.20s
2020-06-08 13:20:30,240 p=24483 u=vagrant n=ansible | - downloading role 'swapfile', owned by oefenweb
2020-06-08 13:20:50,651 p=24483 u=vagrant n=ansible | - downloading role from https://github.com/Oefenweb/ansible-swapfile/archive/v2.0.26.tar.gz
2020-06-08 13:20:52,318 p=24483 u=vagrant n=ansible | - extracting oefenweb.swapfile to /etc/ansible/roles/oefenweb.swapfile
2020-06-08 13:20:52,319 p=24483 u=vagrant n=ansible | - extracting oefenweb.swapfile to /home/vagrant/.ansible/roles/oefenweb.swapfile
2020-06-08 13:20:52,323 p=24483 u=vagrant n=ansible | [WARNING]: - oefenweb.swapfile was NOT installed successfully: the specified
role oefenweb.swapfile appears to already exist. Use --force to replace it.

2020-06-08 13:20:52,324 p=24483 u=vagrant n=ansible | ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.
2020-06-08 13:23:35,433 p=24974 u=vagrant n=ansible | [WARNING]: - oefenweb.swapfile was NOT installed successfully: Unknown error
when attempting to call Galaxy at 'https://galaxy.ansible.com/api/': ('The read
operation timed out',)

2020-06-08 13:23:35,435 p=24974 u=vagrant n=ansible | ERROR! - you can use --ignore-errors to skip failed roles and finish processing the list.
